src/components/validation/runner/config.rs

```diff
@@ -127,7 +135,10 @@ fn build_output_edge() -> (OutputEdge, impl Into<Sinks>) {
     let output_listen_addr = GrpcAddress::from(next_addr());
     debug!(endpoint = %output_listen_addr, "Creating controlled output edge.");
 
-    let output_sink = VectorSinkConfig::from_address(output_listen_addr.as_uri());
+    let mut output_sink = VectorSinkConfig::from_address(output_listen_addr.as_uri());
+    output_sink.batch.timeout_secs = Some(0.1);
+    output_sink.request.retry_attempts = Some(0);
```

Comment:
What's the reasoning behind this change? Not delay sending the batch seems logical, but I'm not sure I get the retry attempts.